infra/modules/vault-access-token/main.tf

```
@@ -10,28 +10,28 @@
 # see : https://developer.hashicorp.com/vault/tutorials/tokens/tokens#periodic-service-tokens
 
 resource "vault_token_auth_backend_role" "lambda_role" {
-  role_name              = var.instance_id
-  allowed_policies       = [var.vault_policy_name]
-  orphan                 = true
-  token_period           = 14*24*60*60 # 2 weeks
-  renewable              = true
+  role_name        = var.instance_id
+  allowed_policies = [var.vault_policy_name]
+  orphan           = true
+  token_period     = 14 * 24 * 60 * 60 # 2 weeks
+  renewable        = true
 
   # no explicit_max_ttl, so should be a 'periodic' token
 
 }
 
 resource "vault_token" "periodic_service_token" {
-  role_name        = vault_token_auth_backend_role.lambda_role.role_name # name to give token's role in Vault
-  period           = "${14*24}h" # 2 weeks
-  renewable        = true
-  renew_min_lease  = 8*24*60*60  # 8 days
-  renew_increment  = 14*24*60*60 # 2 weeks
-  policies         = [var.vault_policy_name]
+  role_name       = vault_token_auth_backend_role.lambda_role.role_name # name to give token's role in Vault
+  period          = "${14 * 24}h"                                       # 2 weeks
```

Too much spaces for this value?
```suggestion
  period          = "${14 * 24}h" # 2 weeks
```

---

must have been `terraform fmt`